components/hab/src/main.rs

```diff
@@ -161,13 +160,11 @@ fn start(ui: &mut UI) -> Result<()> {
 }
 
 fn sub_cli_setup(ui: &mut UI) -> Result<()> {
-    let fs_root = henv::var(FS_ROOT_ENVVAR).unwrap_or(FS_ROOT_PATH.to_string());
-    let fs_root_path = Some(Path::new(&fs_root));
```

Comment:
@fnichol I adressed this by creating a private static ref which encapsulates this behaviour and uses that throughout this module